README.md

```diff
@@ -82,16 +77,15 @@ To set environment variables, create a `.env` file in the root directory and add
 
 ```bash
 NB_API_QUERY_URL=http://localhost:8000/
-NB_IS_FEDERATION_API=false # For node API
 ```
 
-if you're using the remote (in this example federation) api, your `.env` file would look something like this:
+if you're using the remote api, your `.env` file would look something like this:
 
 ```bash
 NB_API_QUERY_URL=https://federate.neurobagel.org/
 ```
 
-if you're using a federation api with authentication, your `.env` file would look something like this:
+if you're using a remote api with authentication, your `.env` file would look something like this:
```

Comment:
in this case, why not keep `federation`?  we now know it has to always be a f-API.

Reply:
Since we only support `federation`, the condition here (if) no longer makes sense.